CHANGELOG.md

```diff
@@ -1,3 +1,130 @@
+- Add `registry_mirrors` to Seed node settings [#6667](https://github.com/kubermatic/kubermatic/issues/6667) 
+- Make the `externalCloudProvider` feature gate impossible to disable once it has been enabled [#6668](https://github.com/kubermatic/kubermatic/issues/6668) 
+- Fix CE installer binary in EE downloads [#6673](https://github.com/kubermatic/kubermatic/issues/6673) 
+- Upgrad Gatekeeper from 3.1.0-beta-9 to 3.1.3. NOTICE: this change also moves the Gatekeeper deployment from the Seed to the User clusters. This means that the user clusters will need some additional resources to run the Gatekeeper Pods. Admins please refer to the upgrade guidelines in the documentation. [#6706](https://github.com/kubermatic/kubermatic/issues/6706) 
+- Change default gatekeeper webhook timeout to 3 sec [#6709](https://github.com/kubermatic/kubermatic/issues/6709) 
+- Add Multus-CNI to accessible addons. [#6731](https://github.com/kubermatic/kubermatic/issues/6731) 
+- Fix S3 storage uploader CA bundle option flag [#6732](https://github.com/kubermatic/kubermatic/issues/6732) 
+- Allow to disable the s3-credentials Secret in the Minio chart [#6760](https://github.com/kubermatic/kubermatic/issues/6760) 
+- Fix installer trying an invalid certificate to test cert-manager [#6761](https://github.com/kubermatic/kubermatic/issues/6761) 
+- Add `enable` and `enforce` OPA options to Admin Settings [#6787](https://github.com/kubermatic/kubermatic/issues/6787)
+- Installer does not listen on port 8080 anymore [#6788](https://github.com/kubermatic/kubermatic/issues/6788) 
+- Node-local-dns is now using UDP for external queries [#6796](https://github.com/kubermatic/kubermatic/issues/6796) 
+- Fix cases where GET and LIST endpoints for Kubermatic Constraints failed or didn't return all results because there were no related synced Gatekeeper Constraints on the user cluster by just taking the Status from the Gatekeeper Constraints and setting the Synced status to false if the Gatekeeper Constraint is missing. [#6800](https://github.com/kubermatic/kubermatic/issues/6800) 
+- Add validation for Kubermatic Constraint Template API. [#6841](https://github.com/kubermatic/kubermatic/issues/6841) 
+- Fetch the provisioning cloud-init over the api-server  [#6843](https://github.com/kubermatic/kubermatic/issues/6843) 
+- Add `FELIX_IGNORELOOSERPF=true` to `calico-node` container env to allow running on nodes with `net.ipv4.conf.*.rp_filter = 2` set. [#6865](https://github.com/kubermatic/kubermatic/issues/6865) 
+- Hetzner AMD Cloud Server (CPX) now selectable when creating a user cluster [#6872](https://github.com/kubermatic/kubermatic/issues/6872)
+- When creating Hetzner Clusters, specifying the network is now mandatory. [#6878](https://github.com/kubermatic/kubermatic/issues/6878) 
+- Add VNet resource group for Azure. [#6908](https://github.com/kubermatic/kubermatic/issues/6908) 
```

Comment:
I don't remember cherry-picking this until now: https://github.com/kubermatic/kubermatic/pull/6937

Can you confirm that this was generated based on the release branch instead of master?

Reply:
Sure